Scripts/developer_scripts/autotest_cgal

```diff
@@ -380,6 +380,8 @@ setup_dirs()
   CGAL_DIR=`readlink "${CGAL_ROOT}/CGAL-I"`
 
   CGAL_TEST_DIR=${CGAL_DIR}/test
+  CGAL_DATA_DIR=${CGAL_DIR}/data
+  export CGAL_DATA_DIR=$(echo "$CGAL_DATA_DIR" | sed -E 's/\/cygdrive\/([a-z])\//\U\1:\//')
```

Comment:
That seems in conflict with #4229. Is that on purpose?

Reply:
It is here to be able to test #4229, yes.